sklearn/inspection/_plot/decision_boundary.py

```diff
@@ -195,6 +215,12 @@ def plot(self, plot_method="contourf", ax=None, xlabel=None, ylabel=None, **kwar
         ylabel : str, default=None
             Overwrite the y-axis label.
 
+        response_method_used : str, default=None
+            If provided, it will be used to create the plot's title.
```

Comment:
I am not sure if I would add this to `plot`.

If you were instantiating the class (not recommended) and then using `plot`, you'd just rely on the `response_method` passed on init.

If you were using `from_estimator` or `from_estimator` it would be inferred.

Reply:
Not sure if this has been addressed?